warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__schedule_downloaded_successfully.sql

```diff
@@ -0,0 +1,37 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('stg_gtfs_guidelines__feed_guideline_index') }}
+    WHERE check = {{ static_feed_downloaded_successfully() }}
+),
+
+gtfs_schedule_fact_daily_feeds AS (
+    SELECT * FROM {{ ref('gtfs_schedule_fact_daily_feeds') }}
+),
+
+static_feed_downloaded_successfully_check AS (
+    SELECT
+        feed_key,
+        date,
+        CASE
+            WHEN extraction_status = "success" THEN "PASS"
+            WHEN extraction_status = "error" THEN "FAIL"
+        ELSE null
```

Comment:
unsure about bigquery, but a lot of databases return `null` if none of the cases match

Reply:
yes, I think that is desirable... I can add a check that it's not null? but I don't want to default in case something weird happens, I think we'd want to investigate